database/kwdb.sql

```diff
@@ -0,0 +1,172 @@
+PRAGMA foreign_keys = 'ON';
+BEGIN TRANSACTION;
+-- Tables
+CREATE TABLE IF NOT EXISTS "command_label" (
+  "id" INTEGER NOT NULL UNIQUE,
+  "name" TEXT NOT NULL UNIQUE,
+  PRIMARY KEY("id")
+);
+
+CREATE TABLE IF NOT EXISTS "config" (
+  "id" INTEGER NOT NULL UNIQUE,
+  "name" TEXT NOT NULL,
+  "description" TEXT,
+  "path" TEXT NOT NULL UNIQUE,
+  PRIMARY KEY("id")
+);
+
+CREATE TABLE IF NOT EXISTS "event" (
+  "id" INTEGER NOT NULL UNIQUE,
+  "date" TEXT NOT NULL,
+  "time" TEXT,
+  PRIMARY KEY("id")
+);
+
+CREATE TABLE IF NOT EXISTS "executed" (
+  "id" INTEGER NOT NULL UNIQUE,
+  "command_label_id" INTEGER NOT NULL,
+  "elapsed_time_in_secs" INTEGER NOT NULL,
+  "status_id" INTEGER NOT NULL,
+  PRIMARY KEY("id"),
+  FOREIGN KEY("id") REFERENCES "event"("id")
+    ON UPDATE CASCADE ON DELETE CASCADE,
+  FOREIGN KEY("command_label_id") REFERENCES "command_label"("id")
+    ON UPDATE CASCADE ON DELETE RESTRICT,
+  FOREIGN KEY("status_id") REFERENCES "status"("id")
+    ON UPDATE CASCADE ON DELETE RESTRICT
```

Comment:
This is to prevent the deletion of a status or command label that has events attached to it. So you can't delete the 'build' command label if you still have 'build' entries in your statistics.